### PR TITLE
adding undead to to qwc trigger and ndb

### PR DESCRIPTION
**Pull request by tsartin**
adding undead to to qwc trigger and ndb. Fixing minor capitalization error. proposed in #809

### CODE REVIEW

File: svo (namedb).xml [hunk 1]
```diff
@@ -763,7 +764,7 @@ for _, name in ipairs(data) do
     if city_colours[rgb] then
       temp_name_list[#temp_name_list].city = city_colours[rgb]
     else
-      temp_name_list[#temp_name_list].city = "rogue"
+      temp_name_list[#temp_name_list].city = "Rogue"
```
Comment (TheLastDarkthorne) [inline svo (namedb).xml]: Apparently this is by design as ndb has it lowercase as well. At least it does not seem to be breaking highlighting and recognizing of rogue people

Comment (tsartin) [reply svo (namedb).xml]: Think I should change it back?

Comment (TheLastDarkthorne) [reply svo (namedb).xml]: yeah should, unless you can confirm it is not breaking ndb on rogues

**Comment by TheLastDarkthorne** [comment]
Just taking advantage to add a few things I noticed were missing!

**Pull request merged**